src/main/scala/tech/cryptonomic/conseil/tezos/BlockOperations.scala

```diff
@@ -0,0 +1,35 @@
+package tech.cryptonomic.conseil.tezos
+import TezosTypes.{
+  Block,
+  Delegation,
+  InternalOperationResults,
+  Operation,
+  OperationsGroup,
+  Origination,
+  Reveal,
+  Transaction
+}
+
+object BlockOperations {
+
+  /**  Utility extractor that collects, for a block, both operations and internal operations results, grouped
+    * in a form more amenable to processing
+    * @param block the block to inspect
+    * @return a Map holding for each group both external and internal operations' results
+    */
+  def extractOperationsAlongWithInternalResults(
```

Comment:
How come this is pulled into a separate object when only one class is using it?

Reply:
Originally I used that in different classes, I think after some refactor and simplification, it's no more needed as an external utility, I'll merge it back as a local method where it's used.